src/MobileKidsIdApp/MobileKidsIdApp/MobileKidsIdApp.UWP/MobileKidsIdApp.UWP.csproj

```diff
@@ -140,6 +140,11 @@
       <Generator>MSBuild:Compile</Generator>
       <SubType>Designer</SubType>
     </ApplicationDefinition>
+    <Page Include="Styles.xaml">
```

Comment:
`Error error WMC9999: Could not find file 'C:\projects\mobilekidsidapp\src\MobileKidsIdApp\MobileKidsIdApp\MobileKidsIdApp.UWP\Styles.xaml'.`